### PR TITLE
Add docstring in Series.rename and Series.alias

### LINKED ISSUE #131: Can't use Spark's .alias() to rename column
I would like to rename a column using .alias() instead of the pandas syntax (it's cleaner in this case), but I get the error message seen below. .toDF() works for renaming all the columns. Can we implement .alias() please?
Screen Shot 2019-04-22 at 7 47 50 AM

### PR DESCRIPTION
**Pull request by HyukjinKwon**
Resolves #131

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
cc @rxin

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -206,8 +206,8 @@ def getField(self, name):
                     "Field {} not found, possible values are {}".format(name, ", ".join(fnames)))
             return Series(self._scol.getField(name), self._kdf, self._index_info)
 
-    # TODO: automate the process here
     def alias(self, name):
+        """An alias for :meth:`Series.rename`."""
```
Comment (rxin) [inline databricks/koalas/series.py]: we should also add alias to the Series doc page.

Two ways:

One is to create a Spark API section, and just put alias there. The other is to put it in the same section as rename. WDYT? cc @thunterdb

**Comment by rxin** [comment]
Merging now. Please submit a follow-up PR for adding alias to the doc page once we decided on which section to put it.

**Comment by HyukjinKwon** [comment]
Will do. Thanks.

File: databricks/koalas/series.py (hunk 1 above)
Comment (thunterdb) [reply databricks/koalas/series.py]: Nothing prevents you from putting it into both sections. We will eventually have a spark section anyway.

Comment (rxin) [reply databricks/koalas/series.py]: Yes that's a brilliant idea Tim.

**Pull request merged**